srcpkgs/up/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'up'
+pkgname=up
+version=0.3
+revision=1
+build_style=go
+go_import_path="github.com/akavel/up"
+hostmakedepends="git"
+short_desc="Ultimate Plumber - write Linux pipes interactively"
+maintainer="Piraty <piraty1@inbox.ru>"
+license="Apache-2.0"
```

Comment:
Missing vlicense in post_install

Reply:
Apache-2.0 doesn't require we ship it, only MIT, ISC, BSD-*